packages/support/lib/image-util.js

```diff
@@ -44,23 +38,23 @@ const MATCH_NEIGHBOUR_THRESHOLD = 10;
 
 const AVAILABLE_DETECTORS = [
   'AKAZE',
-  'AGAST',
+  //'AGAST',
```

Comment:
What was the reason to exclude these detectors? In theory we could have them all in the list and let the code fail if the particular OpenCV deployment does not support it

Reply:
this code will only ever run against opencv-bindings. And I was unable to find these detectors in this package anywhere. so there's no reason to include them if we know they won't work

Reply:
actually I found all except SIFT. their feature detector naming scheme is not consistent.